### ISSUE TITLE
ES6 Class does not honor indent for first indentation level

### ISSUE THREAD

**Issue by whomba** [eslint/eslint#5084]
# Issue: 
When using ES6, a file with class does not honor spacing for first indentation level

# ESLint Rule:
indent: [2, 4]

# What happens:

```js
class testClass {
    testOne(){   // Works as expected
        let foo = 23;  // Works as expected
        foo += 10;  // Works as expected
    }
  
  testTwo(){   // DOES NOT FAIL, it should as it's only indented by 2, not 4
      let bar = 23;   // Works as expected
      bar += 10;   // Works as expected
  } 
}
```

As you can see above, the function ```testTwo``` is indented by two, and it should be indented by 4 yet an error is not 

# Demo:
* Go to http://eslint.org/demo/
* Copy above code in 
* Disable all rules except 'indent' (defaults to 4)
* Everything passes when line 7 should be an error.

In addition, if you enter the following code: 
```
class testClass {
    testOne(){   // Works as expected
        let foo = 23;  // Works as expected
        foo += 10;  // Works as expected
    }
  
testTwo(){   // DOES NOT FAIL, it should as it's not indented at all
      let bar = 23;   // 8:7 - Expected indentation of 4 space characters but found 6. 
      bar += 10;   // 9:7 - Expected indentation of 4 space characters but found 6. 
} 
}
```

Where ```testTwo``` has no spaces this does not throw an error either.

# ESLint Version:
v1.10.3

**Comment by eslintbot** [comment]
@whomba Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Class body support was not present for indent rule in `1.10.3`. It has been added in 2.0.0 beta and will be released with it.

**Comment by whomba** [comment]
Great, thanks @gyandeeps Any word on when 2.0 beta will be rolling out (days / weeks / months / years?)

Thanks!

**Comment by gyandeeps** [comment]
beta is out but there is no firm date on final 2.0 release.

**Comment by whomba** [comment]
Alright, thank you.
